packages/library/src/browser/mapper.library.ts

```
@@ -0,0 +1,200 @@
+
+		geoCoords(coords)																			// get a Location object
+			.then((loc) => {
+				switch (true) {
+					case (!('maps' in window['google'])):
+						throw new Error('Google Maps API not configured');
+
+					case isNullish(loc):															// unsuccessful geoLocation
+						throw new Error('Cannot determine Coordinates');
+
+					case isNullish(coords):													// current location
+						const test1 = !isNullish(mapStore.geolocation) && !isNullish(mapStore.georesponse);
```

Biome (2.4.9)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)